packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -800,26 +996,28 @@ contract DeployImplementations is Script {
 contract DeployImplementationsInterop is DeployImplementations {
     function createOPSMContract(
         DeployImplementationsInput _dii,
-        DeployImplementationsOutput,
-        OPStackManager.Blueprints memory blueprints,
-        string memory release,
-        OPStackManager.ImplementationSetter[] memory setters
+        DeployImplementationsOutput _dio,
+        OPStackManager.Blueprints memory _blueprints,
+        string memory _release,
+        OPStackManager.ImplementationSetter[] memory _setters
     )
         internal
         override
         returns (OPStackManager opsmProxy_)
     {
-        SuperchainConfig superchainConfigProxy = _dii.superchainConfigProxy();
-        ProtocolVersions protocolVersionsProxy = _dii.protocolVersionsProxy();
         ProxyAdmin proxyAdmin = _dii.superchainProxyAdmin();
 
         vm.startBroadcast(msg.sender);
         Proxy proxy = new Proxy(address(msg.sender));
-        OPStackManager opsm = new OPStackManagerInterop(superchainConfigProxy, protocolVersionsProxy);
+
+        deployOPContractsManagerImpl(_dii, _dio); // overriding function
```

Comment:
One thing I don't love is that we `startBroadcast` then jump into a function. So when modifying that `deployOPContractsManagerImpl` the dev hsa to remember that are in a broadcast context, which isn't obvious. I would suggest changing this `vm.startBroadcast` to just `vm.broadcast`, and again using `vm.broadcast` within `deployOPContractsManagerImpl`. Then for the below calls, you can either use multiple `vm.broadcast` calls or bundle them within a start/stopBroadcast

Reply:
Good call, added in:https://github.com/ethereum-optimism/optimism/pull/12035/commits/c131a31a0cd4683f239d8a59df7fc6bd70f271ed